.buildkite/pipeline.yml

```diff
@@ -43,3 +43,31 @@ steps:
         queue:
           - mac
           - default
+
+  - label: ":node: [Test] curlrc option"
+    command: |
+      set -euo pipefail
+
+      echo "--- Verify node installation"
+      cd tests/dummy && npm install
+
+      echo "--- Verify curl output is NOT verbose"
+      curl -s https://google.com
```

Comment:
Isn't this one unneeded, given you also run the post-nvm-install test `curl` call inline in the `if` statement 2 lines below? 🤔

Reply:
Not necessarily needed. Printing the webpage content is just to give us some assurance that the check below is correct, because we can look at its content and see it's not a verbose curl output. Without this line, this "Verify curl output is NOT verbose" step [prints nothing and looks a bit strange](https://buildkite.com/automattic/nvm-buildkite-plugin/builds/57#018b887d-6fe5-41e0-8b6f-f91da2e062a1/588).

Reply:
Oh I see because in the curl call you make within the `if` condition you pipe the output to `| grep` so it swallows the content it filtered out.

That's still a bit of a shame that we end up making the same request two times (instead of eg storing the result in a variable then print that variable then grep on it)… but given the context (a unit test that is not gonna be run that often as I don't expect this repo/plugin to be as active as eg an app repo, + it's the Google home page so shouldn't be that big in size to request twice) probably not worth the effort to optimize for so little, so 🤷‍♂️👍